static/app/views/app.html

```diff
@@ -169,10 +171,26 @@ <h3 class="panel-title">{{service.entity.name}}</h3>
             </div>
             <div class="panel-body">
                 <button type="button" class="btn btn-xs btn-info" ng-disabled="disableServiceBinder"
-                ng-show="!service.boundService" ng-click="bindService(service)">Bind Service to App</button>
-                <button type="button" class="btn btn-xs btn-warning" ng-disabled="disableServiceBinder"
-                ng-show="service.boundService" ng-click="unbindService(service)">Unbind Service to App</button>
+                ng-show="!service.boundService" mwl-confirm title="Notice!" message="You will need to <b>restage</b> this application before this service is available."
+                on-confirm="bindService(service)" confirm-button-type="info" cancel-button-type="default">Bind Service</button>
+                <button type="button" class="btn btn-xs btn-danger" ng-disabled="disableServiceBinder"
+                ng-show="service.boundService" mwl-confirm title="Warning!" message="Are you sure you want to unbind this service?"
+                on-confirm="unbindService(service)" confirm-button-type="danger"
+                cancel-button-type="default">Unbind Service</button>
+            </div>
+            <div class="panel-heading" ng-show='service.credentials'>
+                <h3 class="panel-title">VCAP Info</h3>
+            </div>
+            <div class="panel-body" ng-show='service.credentials' ng-init='showCredentials=false'>
+                <button type="button" class="btn btn-xs btn-danger" ng-hide="showCredentials" mwl-confirm
+                title="Warning!" message="These are senstive data DO NOT SHARE!"
```

Comment:
Let's be more specific with the message.
"This info may contain sensitive data such as passwords. DO NOT SHARE!"

I kind of want to say something about watching your surroundings before clicking but not sure how to say it concisely.